src/Pages/Coins.js

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import Coin from '../components/coin/Coin';
+/* eslint-disable */
+import { fetchCoins } from '../redux/coins/coinsReducer';
+import './Coins.css';
+/* eslint-enable */
```

Comment:
- [ ] Here is the same issue as I explained in the above comments, make sure you also fix it:
```JavaScript
// Bad
import { fetchCoins } from '../redux/coins/coinsReducer';

// Good
import { fetchCoins } from '../Redux/coins/coinsReducer';
```